python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -352,6 +356,48 @@ def __new__(
         )
 
 
+@whitelist_for_serdes
+@experimental
+class BackfillRequest(
+    NamedTuple(
+        "_BackfillRequest",
+        [
+            ("asset_partitions", "AssetGraphSubset"),
+            ("tags", PublicAttr[Optional[Mapping[str, str]]]),
+        ],
+    )
+):
+    @property
+    def asset_keys(self) -> AbstractSet[AssetKey]:
+        return self.asset_partitions.asset_keys
+
+    @classmethod
+    def from_asset_selection_and_partitions(
```

Comment:
What about `from_assets_partitions`?

A future method with the same signature that targets downstreams could be called `from_assets_partitions_with_downstreams`

Reply:
I think there's a decent chance in the future that we'll add a static constructor named `from_asset_partitions` that accepts a list of `AssetKeyPartitionKey`s. Wary that having both `from_asset_partitions` and `from_assets_partitions` might be confusing.